### PR TITLE
Remove publish via http field since it's no longer valid

### PR DESCRIPTION
**Pull request by latran**
This field was removed from previous sat version.  Need to check with dev if "unprotected" was its replacement or not.

### CODE REVIEW

**Comment by latran** [comment]
Confirmed with dev that "Publish via HTTP" is replaced with "Unprotected."

See [PR#10094](https://github.com/SatelliteQE/robottelo/pull/10094) for test run result

File: airgun/views/repository.py [hunk 1]
```diff
@@ -274,7 +270,6 @@ class FileRepository(View):
         upstream_url = EditableEntry(name='Upstream URL')
         verify_ssl = EditableEntryCheckbox(name='Verify SSL')
         upstream_authorization = AuthorizationEntry(name='Upstream Authorization')
-        publish_via_http = EditableEntryCheckbox(name='Publish via HTTP')
         unprotected = EditableEntryCheckbox(name='Unprotected')
```
Comment (Griffin-Sullivan) [inline airgun/views/repository.py:273]: Are file and yum repos the only repository types that support unprotected?

Comment (Griffin-Sullivan) [reply airgun/views/repository.py:273]: Just checked 6.12 and they are the only repo types with unprotected as an option.

**Review by Griffin-Sullivan — APPROVED** [review]

**Review by swadeley — APPROVED** [review]
ACK

**Pull request merged**